Documentation: fix typo, add to features list (#4624)
paperless-ngx/paperless-ngx: docs/index.md

## Features

-   **Organize and index** your scanned documents with tags, correspondents, types, and more.

> -   Performs **OCR** on your documents, adding selectable text to image-only documents and makes their content searchable.

Reviewer: ```suggestion
-   Performs **OCR** on your documents, adding searchable and selectable text, to even documents scanned with only images.
```

>     - Paperless utilizes the open-source Tesseract engine to recognize more than 100 languages.

Reviewer: ```suggestion
-   Utilizes the open-source Tesseract engine to recognize more than 100 languages.
```

> -   All documents are saved as PDF/A, a special file format intended for long term storage, alongside the unaltered originals.

Reviewer: ```suggestion
-   Documents are saved as PDF/A format which is designed for long term storage, alongside the unaltered originals.
```
